lib/snapshot.js

```diff
@@ -52,7 +52,9 @@ var default_UA         = '';
 
 async function init_browser() {
 	const args = process.env.MSHOTS_CONTAINERIZED ? [ '--no-sandbox' ] : [];
-	browser = await _puppeteer.launch( { headless: true, timeout: 30000, ignoreHTTPSErrors: true, args } ).
+  const executablePath = process.env.ARCH === "arm64" ? "chromium" : undefined;
```

Comment:
Passing `undefined` does restore the already-working code. The actual (online) code is not 100% exactly correct. But the system has been working well for months even with "complex" sites, so IMHO is better not to change it. It's not part of this particular PR. Maybe in the future, we can consider adding a new PR. It's just a performance upgrade and nothing more.